include/deal.II/matrix_free/operators.h

```diff
@@ -828,29 +1194,28 @@ namespace MatrixFreeOperators
   Base<dim,VectorType>::vmult (VectorType       &dst,
                                const VectorType &src) const
   {
-    typedef typename Base<dim,VectorType>::value_type Number;
```

Comment:
why this change? We don't use vectors this way now, but I think your code won't work with `std::complex<double>`.